### PR TITLE
Anomaly #9976 UMR: undefined service method

### PR DESCRIPTION
**Pull request by ego-axelor**

### CODE REVIEW

File: axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  public String getUmrNumber(Partner partner) {
+    return getUmrNumber(partner, false);
+  }
+
+  @Override
+  public String getUmrNumber(Partner partner, boolean isRecoveryContractLine) {
+
+    String rumNumber = "";
+
+    if(isRecoveryContractLine) {
```
Comment (gdu-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java]: isRecovery is enough. Here contractLine has no sense.

File: axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      rumNumber += "++";
+    }
+
+    rumNumber += partner.getId();
```
Comment (gdu-axelor) [inline axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java]: Better to use partner.getPartnerSeq here

File: axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java (hunk 1 above)
Comment (ego-axelor) [reply axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java]: done

File: axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java (hunk 2 above)
Comment (ego-axelor) [reply axelor-account/src/main/java/com/axelor/apps/account/service/umr/UmrNumberServiceImpl.java]: done

**Pull request merged**